app/helpers/visns_helper.rb

```diff
@@ -3,7 +3,7 @@ module VisnsHelper
 
   def get_facility_locations_by_visn(visn)
     sorted_facility_locations = VaFacility.get_by_visn(visn).get_locations
```

Comment:
nit: you can chain `.sort` onto the end of line 5

Reply:
I changed it.